notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb

```diff
@@ -87,20 +87,23 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    if not Path("251036317-551a2399-303e-4a4a-a7d6-d7ce973e05c5.png").exists():
</pre><p>o we really want to preserve this long and automatically generated name? download_file allows to specify file name for saving</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
renamed